src/components/modules/toolbar/conversion.ts

```diff
@@ -68,7 +68,7 @@ export default class ConversionToolbar extends Module {
     this.nodes.tools = $.make('div', ConversionToolbar.CSS.conversionToolbarTools);
 
     const label = $.make('div', ConversionToolbar.CSS.conversionToolbarLabel, {
-      textContent: 'Convert to',
+      textContent: I18n.t('ui.inlineToolbar.converter', 'Convert to'),
```

Comment:
I think we should move all internal namespaces and messages to typed objects or enums